src/components/20-molecules/dropdown/index.js

```diff
@@ -439,18 +507,46 @@ class AXADropdown extends NoShadowDOM {
   }
 
   firstUpdated() {
+    this.button = this.querySelector('.js-dropdown__toggle');
     this.dropdown = this.querySelector('.js-dropdown__content');
     this.select = this.querySelector('.js-dropdown__select');
     window.addEventListener('resize', this.handleResize);
     window.addEventListener('click', this.handleWindowClick);
     window.addEventListener('axa-dropdown-close', this.handleWindowClick);
   }
 
-  updated() {
+  // build one long string of words paired with their indices in `items`,
+  // separating the elements in each pair with suitable control characters
+  // that do not occur in typed input
+  buildAutosuggestDictionary() {
+    this._autosuggestLetters = new Set();
+    const { items, _autosuggestLetters: letters } = this;
+
+    this._autosuggestDictionary =
+      items
+        .filter(item => !item.disabled)
+        .map((item, index) => {
+          // normalize word for purposes of matching
+          const word = item.name.toLowerCase();
+          // construct a set of all letters in word
+          // (used for an early filter over keystrokes)
+          word.split('').forEach(letter => {
+            letters.add(letter.toLowerCase());
```

Comment:
isn't everything already lowercase?

Reply:
No, it is not. `item.name` can be an arbitrary string, so I lowercase it here and then also lowercase the user input during autosuggest, so they can match up regardless of case differences.